Simplified/NYPLReaderTOCViewController.m

```diff
@@ -163,7 +163,7 @@ - (UITableViewCell *)tableView:(__attribute__((unused)) UITableView *)tableView
       }
       return cell;
     }
-    case 1:{
+    default: {
```

Comment:
Our TOC has two segments right now, and if we have the default being the "2nd" segment, then this would turn into a bug if that segment control ever changes by another developer. Since for now we essentially want to quickly find this as an error, we could actually invoke a crash in the default case by throwing an exception:
```
default:
  @throw NSInvalidArgumentException;
```

Reply:
Seconding @gioneill on this one. Even better would be to have a function of type `(NSInteger) -> SegmentedControlType` where `SegmentedControlType` is an enum we defined. We could then switch on the result of applying the function to the integer and only have to handle cases explicitly defined, thus obviating the need for an unsafe default case here and making the integer check explicit in the conversation function.